ras/uprobe.py

```diff
@@ -36,27 +35,61 @@ def run_cmd(self, cmd):
 
     @staticmethod
     def run_cmd_out(cmd):
-        return process.system_output(cmd, shell=True, ignore_status=True,
+
+        """
+        Run the linux terminal command
+        Ex: cat, ldd ..etc.
+        """
+        return process.system_output(cmd,
+                                     shell=True, ignore_status=True,
                                      sudo=True).decode("utf-8")
 
     def clear_trace(self):
-        self.debugfs = "/sys/kernel/debug/tracing/"
-        self.tracefs = os.path.join(self.debugfs, "trace")
-        self.uprobes_enable_fs = os.path.join(self.debugfs, "events/uprobes/enable")
-        self.uprobes_event_fs = os.path.join(self.debugfs, "events/uprobes/u_malloc/enable")
-        self.uprobes_events_fs = os.path.join(self.debugfs, "uprobe_events")
-        genio.write_one_line(self.tracefs, "")
-        genio.write_one_line(self.uprobes_events_fs, "")
-
-    def enable_uprobes(self):
-        genio.write_one_line(self.uprobes_enable_fs, "1")
-        genio.write_one_line(self.uprobes_event_fs, "1")
-
-    def disable_uprobes(self):
-        genio.write_one_line(self.uprobes_event_fs, "0")
-        genio.write_one_line(self.uprobes_enable_fs, "0")
+
+        """
+        Clear the tracepoint and uprobe events
+        before enable and disable the probe.
+        """
+
+        try:
+            self.debugfs = "/sys/kernel/debug/tracing/"
+            self.tracefs = os.path.join(self.debugfs, "trace")
+            uprobes_en = "events/uprobes/enable"
+            self.uprobes_enable_fs = os.path.join(self.debugfs, uprobes_en)
+            uprobes_umall = "events/uprobes/u_malloc/enable"
+            self.uprobes_event_fs = os.path.join(self.debugfs, uprobes_umall)
+            uprobe_eve = "uprobe_events"
+            self.uprobes_events_fs = os.path.join(self.debugfs, uprobe_eve)
+            genio.write_one_line(self.tracefs, "")
+            genio.write_one_line(self.uprobes_events_fs, "")
+        except IOError:
+            self.cancel("There is issue with kernel system resources")
+
+    def ena_dis_uprobes(self, prob_value):
+        """
+        Enable and Disable the uprobes.
+        ------------
+        Enable_uprobes:
+        prob_value = 1
+        ------------
```

Comment:
I feel these cosmetic line are not required here as your already adding new line for better visualization.